devops/prod/values.yaml

```
@@ -111,6 +111,7 @@ iam-cache-server-helm:
     JWT_PRIVATE_KEY_PATH: /keys/private.pem
     JWT_PUBLIC_KEY_PATH: /keys/public.pem
     STRATEGY_CACHE_SERVER: https://identitycache.energyweb.org/v1
+    BLOCKNUM_AUTH_ENABLED: false
```

Should be enabled in prod to not break existing apps. We can disable once we've had time to migrate all of the clients.
```suggestion
    BLOCKNUM_AUTH_ENABLED: true
```